perses/app/setup_relative_calculation.py

```diff
@@ -328,7 +324,7 @@ def run_setup(setup_options):
         _logger.info(f"\tatom selection detected: {atom_selection}")
     else:
         _logger.info(f"\tno atom selection detected: default to None.")
-        atom_selection = None
+        atom_selection = 'all' 
```

Comment:
Looks like this has changed for some reason?

Reply:
Yes - I don't think saving no atom coordinates should be the default